sonaranalyzer-dotnet/rspec/cs/S3925_c#.html

```diff
@@ -3,17 +3,18 @@
 <p>This rules raises an issue on types that implement <code>ISerializable</code> without following the serialization pattern recommended by
 Microsoft.</p>
 <p>Specifically this rule checks for these problems:</p>
-<ul>
-  <li> The <code>System.SerializableAttribute</code> attribute is missing. </li>
-  <li> Non-serializable fields are not marked with the <code>System.NonSerializedAttribute</code> attribute. </li>
-  <li> There is no serialization constructor. </li>
-  <li> An unsealed type has a serialization constructor that is not <code>protected</code>. </li>
-  <li> A sealed type has a serialization constructor that is not <code>private</code>. </li>
-  <li> An unsealed type has a <code>ISerializable.GetObjectData</code> that is not both <code>public</code> and <code>virtual</code>. </li>
-  <li> A derived type has a serialization constructor that does not call the <code>base</code> constructor. </li>
-  <li> A derived type has a <code>ISerializable.GetObjectData</code> method that does not call the <code>base</code> method. </li>
-  <li> A derived type has serializable fields but the <code>ISerializable.GetObjectData</code> method is not overridden. </li>
-</ul>
+<p> * The <code>System.SerializableAttribute</code> attribute is missing.</p>
```

Comment:
I think you can manually revert the changes from `<ul><li>` to `<p> *` - this is a bug in the rule-api which I believe is reported (I think I did it) somewhere

The only relevant change in this file is at lines 15-17 (`Classes which inherit from <code>Exception</code>...`)